guide/17-geocoding-guides/part6_custom_geocoders.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Developers integrating the geocoder into their application may need to know the appropriate input field names to use for  &nbsp;the language and country of their users. This information can be obtained using the <code>localizedNames</code> parameter of the  &nbsp;address field. More information on this, as well as on the <code>recognizedNames</code> parameter, is available in the Foreign language  &nbsp;field names (<code>World Geocoding Service</code>) <a href="https://developers.arcgis.com/rest/geocode/api-reference/geocoding-localized-input-field-names.htm#ESRI_SECTION1_420C0A38D36546EDB1FC114048757B3E" target="_blank">documentation</a>. </p><p>For example, the code below lists the supported address fields and the corresponding input field names in <code>Hindi</code> and/or Simplified Chinese:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>